lldb/source/API/SBTarget.cpp

```diff
@@ -255,6 +255,7 @@ SBProcess SBTarget::LoadCore(const char *core_file, lldb::SBError &error) {
     ProcessSP process_sp(target_sp->CreateProcess(
         target_sp->GetDebugger().GetListener(), "", &filespec, false));
     if (process_sp) {
+      ElapsedTime loadCoreTime(target_sp->GetStatistics().GetLoadCoreTime());
```

Comment:
Instead of doing this in two places, why not putting this into `Process::LoadCore()` in case there is a 3rd code path invoking it in future.